pkg/version/channels.go

```diff
@@ -44,6 +45,10 @@ func (c Channels) Match(actualVersion string) error {
 		return errors.New("actual version is empty")
 	}
 
+	if len(c.array) == 0 {
```

Comment:
nit: could probably replace with `if c.Empty()`. 

It should be fine as is, but in the future if we'd change what it means for a channel to be `Empty`, we'd have two places to modify instead of one. Always easier to go through the public interface imo if it's defined. That being said, I doubt this will change much in the future :)